server/src/core/message/message.ts

```diff
@@ -1,22 +1,53 @@
 import { User } from '../user';
 import { extractFirstName } from '../util';
 
 type MessageType = 'sms' | 'email';
 
-export function createDistributionReportSmsMessage(report: DistributionReport, donor: User, beneficiaries: User[], donateLink: string): string {
-  return `Hello ${extractFirstName(donor.name)}, Ksh ${report.totalDistributedAmount} has been transferred from your SocialRelief donation to ${beneficiariesAndAmountReceived(beneficiaries, report.receivedAmount, 'sms')}. Thank you for your contribution. To donate again, click ${donateLink}`;
+export function createDailyDistributionReportSmsMessage(report: DistributionReport, donor: User, beneficiaries: User[], donateLink: string): string {
+  return `Hello ${extractFirstName(donor.name)}, Ksh ${report.totalDistributedAmountFromDonor} has been transferred from your SocialRelief donation to ${beneficiariesAndAmountReceived(beneficiaries, report.receivedAmount, 'sms')}. Thank you for your contribution. To donate again, click ${donateLink}`;
 }
 
-export function createDistributionReportEmailMessage(report: DistributionReport, donor: User, beneficiaries: User[], donateLink: string): string {
+export function createDailyDistributionReportEmailMessage(report: DistributionReport, donor: User, beneficiaries: User[], donateLink: string): string {
   return `<p>
-              Hello ${extractFirstName(donor.name)}, <br><br>
-              Ksh ${report.totalDistributedAmount} has been transferred from your SocialRelief donation to:<br>
-              ${beneficiariesAndAmountReceived(beneficiaries, report.receivedAmount, 'email')}.<br>
-              Thank you for your contribution.<br>
-              To donate again, click <a href='${donateLink}' target="_blank">here</a><br><br>
-              Regards,<br>
-              Social Relief Team
+            Hello ${extractFirstName(donor.name)}, <br><br>
+            Ksh ${report.totalDistributedAmountFromDonor} has been transferred from your SocialRelief donation to:<br>
+            ${beneficiariesAndAmountReceived(beneficiaries, report.receivedAmount, 'email')}.<br>
+            Thank you for your contribution.<br>
+            To donate again, click <a href='${donateLink}' target="_blank">here</a><br><br>
+            Regards,<br>
+            Social Relief Team
+          </p>`;
+} 
+
+export function createMonthlyDistributionReportSmsMessageForContributingDonor(report: DistributionReport, donor: User, beneficiaries: User[], totalDonationsFromAllDonors: number, donateLink: string): string {
+  return `Hello ${extractFirstName(donor.name)}, A total of Ksh ${totalDonationsFromAllDonors} was transferred last month, of which Ksh ${report.totalDistributedAmountFromDonor} came from your SocialRelief donation and went to ${report.beneficiaries.length} beneficiaries, including ${beneficiariesAndAmountReceived(beneficiaries, report.receivedAmount, 'sms')}. Thank you for your contribution. To donate again, click ${donateLink}`;
+}
```

Comment:
I think this message could be shortened while keeping the same content.

E.g.:

```
Hello John, last month a total of Ksh 40000 was donated by 4 Social Relief donors to 10 beneficiaries. Ksh 5000 was sent from your donations to James (2000), John (1000), Mary (2000). Thank you for your contribution. To donate again, click $donateLink
```

Reply:
If the person did not donate, then you can omit the part about how their donation was used.